README.md

```diff
@@ -9,64 +9,75 @@ Richoux, F. and Usunier, N., 2016. _TorchCraft: a Library for Machine Learning R
 on Real-Time Strategy Games_ - [arXiv:1611.00625](https://arxiv.org/abs/1611.00625).
 
 
-## [Installation](docs/user/installation.md)
+## [Installation](docs/user/starcraft_in_windows.md)
 
-Please follow the [installation instructions](docs/user/installation.md). 
+Please follow the [installation instructions](docs/user/starcraft_in_windows.md). 
 
 You will need to have a license of StarCraft: Brood War. 
 
-The most tested and stable way to set it up is to have StarCraft running on Windows 
-and Torch running in Linux or Mac OS X. If you already have StarCraft, you need to
-install [BWAPI](docs/user/installation.md#bwapi),
-[copy a few files](docs/user/installation.md#torchcraft-common-prerequisites), 
-and either launch the [AIModule (DLL)](docs/user/installation.md#torchcraft-aimodule-dll-for-users) 
-OR launch the [AIClient (EXE)](docs/user/installation.md#torchcraft-aiclient-exe-for-users). 
+TorchCraft is proxy into BWAPI, that enables data from StarCraft to be sent 
+over a ZMQ connection so that it can be parsed anywhere. The TorchCraft client 
```

Comment:
Proposed: "TorchCraft is a BWAPI module that sends StarCraft data out over a ZMQ connection. This lets you parse StarCraft data and interact with BWAPI from anywhere."